src/index.svelte

```diff
@@ -53,7 +55,7 @@
         constants.paths.launcherDir.then((launcherDir) => {
             Neutralino.filesystem.getStats(`${launcherDir}/logs/latest.log`)
                 .then(async () => {
-                    let created_at = (await Neutralino.os.execCommand(`stat -c '%W' "${path.addSlashes(`${launcherDir}/logs/latest.log`)}"`)).stdOut;
+                    let created_at = Number.parseInt((await Neutralino.os.execCommand(`stat -c '%W' "${path.addSlashes(`${launcherDir}/logs/latest.log`)}"`)).stdOut);
```

Comment:
Wouldn't this break created_at as it uses the full date format as seen on line 59

Reply:
> Wouldn't this break created_at as it uses the full date format as seen on line 59

- `stat %W` always outputs integer
- `Number.parseInt` returns NaN if input is not a number, and never throw error
- see L61, L63, original code directly assert that `created_at` is a number, so the current way is undoubtedly safer.

so it's fine